_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+
+Therefore, for new code produced by the Hydra-in-a-Box effort, we have first sought to locate it where it rightfully belongs in the stack (e.g., Sufia, an independent gem, or even Fedora). Our hope is that the work we’re doing is general and modular enough that other Hydra efforts can and will make use of our code components--e.g., it would be excellent for [the Avalon Media System](http://www.avalonmediasystem.org/) to make use of (and contribute back to!) our components when it needs similar functionality. 
+
+3. **turnkey repository product,** locally managed. Building on what the Hydra community has learned working on Sufia to make it easy to install and maintain, we expect to provide a complete, configurable, turnkey product--more akin to Avalon, DSpace, or even ContentDM. We need a name for this product analogous to those systems, and with broad market appeal and strong branding identity. We have put a lot of thought into requirements for a good name: 
+
+    * We want something clearly related to the Hydra project to leverage its brand, and to invite uptake and code contributions / maintenance. Further, we want adopters of the product to know it is part of the Hydra community effort, in order to provide an on-ramp for them to join / participate in the wider community effort. 
+
+    * We want something clearly distinct from the Hydra community & repository / application framework. (Just calling it "Hydra" wouldn’t work.)
+
+    * Hydra-in-a-Box and HyBox both seem poor choices here: the former is clunky and overly long. The latter is perhaps too insider-y and lacks the natural power and mass-market appeal that a name like "Avalon" brings.
+
+    * Also, Hydra-in-a-Box implies the full functionality and flexibility of the Hydra technical framework is present in the product, but all nicely wrapped up in one package. This is not true, as we’ll be making opinionated selections, based on the community’s requirements, about functionality to include. As such, we’re looking for a name that connotes one is getting a particular flavor or profile of Hydra.
+
+    * We may want to pick up on the useful trend of picking a name related to or derived from "Hydra" in some way, but distinct. Hydrangea, Hypatia, Hydraulics, Hyperspace all fall into this pattern. 
+
+    * We need to be sure to clear the obvious hurdles--no similar products with similar names, no unintentionally offensive meanings in other languages, available domain names, etc.
+
+In short, we’re looking for a product name **that has strong branding potential, will have mass market appeal, and is distinct from but is easily related to Hydra. **
```

Comment:
You may have to switch the ending `**` to right after "Hydra," before the ending period. The bold is running on.